nuwebots/controllers/nugus_controller/nugus_controller.cpp

```diff
@@ -272,7 +272,7 @@ class PlayerServer {
         const double* Rwx  = robot->getFromDef("BLUE_1")->getField("rotation")->getSFRotation();
 
         // Rotation is an angle axis so convert it to a rotation matrix
-        Hxw.linear() = Eigen::AngleAxisd(Rwx[3], Eigen::Vector3d(Rwx[0], Rwx[1], Rwx[2])).toRotationMatrix().inverse();
+        Hxw.linear() = Eigen::AngleAxisd(Rwx[3], Eigen::Vector3d(Rwx[0], Rwx[1], Rwx[2])).toRotationMatrix();
```

Comment:
Should the name be swapped to Rxw if this is the case?
Lower down on L765, inverse is used. It's all the same maths, so does that need to be changed too?
@Tom0Brien

Reply:
Have you tried with both changed? I guess here it doesn't do much since we  probably start pretty much on identity?

Reply:
Yeah, should change both.

Reply:
You're right, I didn't notice before because I was starting with robot upright, can start anywhere with any orientation if both changed.

Reply:
Updated